apiv2/schema/schema.yaml

```diff
@@ -1771,6 +1780,100 @@ classes:
         unit:
           symbol: °
           descriptive_name: degrees
+  PerSectionParameters:
```

Comment:
@uermel can you take a look at the units and descriptions for these fields and let me know if they make sense.